### ISSUE TITLE
The results on Human3.6M

### ISSUE THREAD

**Issue by luzzou** [chaneyddtt/Generating-Multiple-Hypotheses-for-3D-Human-Pose-Estimation-with-Mixture-Density-Network#5]
Hi, I ran the code by its default setting which is training on Human3.6M, the learning rate and other parameters are remained as the default value. However, I can only achieve 57.75mm MPJPE (with prior loss) after 200 epochs of training, which is different from the reported 52.7mm in the paper. Could you give any suggestions to fix this?

**Comment by luzzou** [comment]
@chaneyddtt  Sorry, I have used the wrong dataset, the problem has be solved!

**Comment by cuge1995** [comment]
> @chaneyddtt Sorry, I have used the wrong dataset, the problem has be solved!

hi, I can't find this dataset, can you give me a web address ？
